README.md

```diff
@@ -1,17 +1,38 @@
-# Sample
+﻿# ABOUT
 
-This module is for fun and trying out new crazy and wacky things with the game. It houses a number of example blocks and UI screens.
+Sample.
 
-### Credits for images:
+This module is for fun and trying out new crazy and wacky things with the game. 
+It houses a number of example blocks and UI screens.
 
-WhiteStone: http://www.planetminecraft.com/texture_pack/new-start---by-loufisch/
+ 
+A person can test their UI screens and other various creations in this module.
+This module is open to local developments and tests.
+It is more inclined towards developers who want to test stuff; but players can enjoy it too!
+
+# LICENSE
+
+The module is licensed under Apache 2.0 license.
+
+# CREDITS
+
+Credits for images:
 
-CementStone: Dane Claessen (https://github.com/Dane99)
+WhiteStone: http://www.planetminecraft.com/texture_pack/new-start---by-loufisch/
```

Comment:
Every single line does not have to start with `Credits for` : That is what the heading implies - that we are giving credits. Revert this, for every single contributor.